README.md

```diff
@@ -29,13 +29,22 @@ $ docker-compose up
 ```
 
 If you are happy building from source you can use the scripts in the
-[`bin/`] directory of this repository. E.g. to start a collector
-pointing to an existing local redis server:
-
-```
-$ git clone https://github.com/openzipkin/zipkin
-$ cd zipkin
-$ ./bin/collector-redis
```

Comment:
I took this out, as the simplest start is to use SQLite (as it doesn't need an external process running). Ack that until tonight, that didn't work :)